knex/knex.js

```diff
@@ -0,0 +1,3 @@
+const environment = process.env.MODE_ENV || 'development'
```

Comment:
do we need this file? Or could this be incorporated into the db file / main knex instance?

Reply:
Not sure! Just followed the instructions for creating seed and migration files.